client/src/pages/About/About.css

```diff
@@ -0,0 +1,46 @@
+html, body {
+    height: 100%;
+    font-family: Cabin;
+}
+
+.container-fluid {
+    height: 600px;
+    background-image: url(../../assets/uclaheader.png);
+    /* This is what makes the background image rescale based on the container's size */
+    background-size: cover;
+
+    /* Background image is centered vertically and horizontally at all times */
+    background-position: center center;
+  
+    /* Background image doesn't tile */
+    background-repeat: no-repeat;
+    
+    /* Background image is fixed in the viewport so that it doesn't move when 
+        the content's height is greater than the image's height */
+    background-attachment: fixed;
+    
+    /* Set a background color that will be displayed
+        while the background image is loading */
+    background-color: #464646;
+    background-color: rgba(244, 207, 76, 0.4);
+    background-blend-mode: multiply;
+}
```

Comment:
When you make changes to the bootstrap class, the changes are visible anywhere we use the bootstrap class. So if you look at the home page or stories page, they now have a parallax effect. 

Instead of styling the bootstrap class, give the element a second class and add the styles there.